fastparse/src-3/fastparse/package.scala

```diff
@@ -0,0 +1,581 @@
+import fastparse.internal.{Instrument, Logger, *}
+
+import language.experimental.macros
+
+package object fastparse {
```

Comment:
package object are deprecated in scala 3. It seems thier support will be  [dropped in 3.2](https://dotty.epfl.ch/docs/reference/dropped-features/package-objects.html) . I think it would be nice to not have to have another migration to be done for a futur scala 3 version.

Reply:
Migrated to top-level functions ✅